radio/src/gui/colorlcd/CMakeLists.txt

```diff
@@ -79,6 +79,10 @@ endmacro(add_gui_src)
 
 if(LUA)
   add_gui_src(standalone_lua.cpp)
+  set(SRC ${SRC} lua/widgets.cpp)
+endif()
+
+if(LUA STREQUAL YES)
```

Comment:
We should scrap this one and place it in the block above. The `if(LUA)` will work for anything like YES, ON, etc.